docs/source/parameter_handling.rst

```diff
@@ -39,4 +39,51 @@ The parameters can then be accessed as
     [('param1', 7.0), ('param2', 4.0)]
     >>> p.exists("param1")
     True
+    
+    
+.. code-block:: python
+ 
+    # print all data inside Param
```

Comment:
```suggestion
    # print the key and value pairs stored in a Param object
```

Reply:
hmm seems that the outdated suggestions can't be applied via the GitHub interface. I think you need to do that manually